### DISCUSSION TITLE
posts/ubuntu22-install-clang10/

### DISCUSSION THREAD

**Discussion by giscus** [leegical/leegical.github.io#6] [Announcements]
# posts/ubuntu22-install-clang10/

本文记录了Ubuntu 22.04 LTS 64位系统安装 clang10 版本的过程。

https://www.haoyep.com/posts/ubuntu22-install-clang10/

**Comment by code33** [comment]
看你的博客很赞的说,请问用的啥程序啊?
感谢

**Reply by leegical** [reply]
- 静态博客编译工具hugo
- 主题[Fixit](https://fixit.lruihao.cn/zh-cn/documentation/getting-started/quick-start/)
- 本博客的大致搭建过程可以参考[这篇文章](https://www.haoyep.com/posts/windows-hugo-blog-github/)

**Comment by xieyi4650** [comment]
非常感谢, 解决了我的问题, 一直apt-get install clang-9不成功,
提示  E: Package 'clang-9' has no installation candidate
